src/js/global.js

```diff
@@ -156,6 +157,20 @@ ubuntu.globalNav = function() {
           }(moreList)
         );
       }
+
+      let smallScreenToggle = globalNav.querySelector('.global-nav__title');
+      let navList = globalNav.querySelector('.global-nav__list');
+      if (smallScreenToggle && navList) {
+        smallScreenToggle.addEventListener(
+          'click',
+          function(smallScreenToggle) {
+            return function(clickEvent) {
+              navList.classList.toggle('is-revealed');
+              smallScreenToggle.classList.toggle('is-revealed');
+            };
+          }(smallScreenToggle)
+        );
+      }
     },
```

Comment:
With this code if you were on a desktop browser and toggle the list, it'll stay open from small screen to large and can not resize. Should we think about checking if the screen is being resized and remove any toggles?

<img width="1119" alt="screen shot 2017-05-02 at 16 08 10" src="https://cloud.githubusercontent.com/assets/1201618/25624590/c15bb906-2f51-11e7-8b2e-cec5fb9f33e7.png">